pkg/tsdb/prometheus/json_to_dataframes_test.go

```diff
@@ -0,0 +1,258 @@
+package prometheus
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+	"github.com/grafana/grafana-plugin-sdk-go/experimental"
+	"github.com/prometheus/client_golang/api"
+	apiv1 "github.com/prometheus/client_golang/api/prometheus/v1"
+	"github.com/stretchr/testify/require"
+)
+
+type MockedRoundTripper struct {
```

Comment:
We should put this stuff underneath the `TestXXX` functions. The idea is that the first thing we see when we enter the file is the most important.

Reply:
done